src/server/scripts/EasternKingdoms/Karazhan/boss_prince_malchezaar.cpp

```diff
@@ -455,9 +454,7 @@ class boss_malchezaar : public CreatureScript
                             if (target)
                             {
                                 axe->AI()->AttackStart(target);
-                                //axe->getThreatManager().tauntApply(target); //Taunt Apply and fade out does not work properly
-                                                                // So we'll use a hack to add a lot of threat to our target
-                                axe->AddThreat(target, 10000000.0f);
+                                AddThreat(target, 10000000.0f, axe);
```

Comment:
None of the changes in these very old hack scripts are sniff sourced (or even tested, for that matter).